Don't install .info files for they haven't been generated by the time
they're installed (see comment in Makefile).

--- doc/CMakeLists.txt.orig	Thu Aug 11 23:12:09 2016
+++ doc/CMakeLists.txt	Thu Aug 11 23:13:41 2016
@@ -122,17 +122,3 @@ if (CMAKE_INSTALL_MANDIR)
   install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ledger.1
     DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 COMPONENT doc)
 endif(CMAKE_INSTALL_MANDIR)
-
-foreach(file ${ledger_doc_files})
-  get_filename_component(file_ext ${file} EXT)
-
-  if(file_ext STREQUAL ".info")
-    if(CMAKE_INSTALL_INFODIR)
-      install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${file}
-        DESTINATION ${CMAKE_INSTALL_INFODIR} COMPONENT doc)
-    endif()
-  elseif(CMAKE_INSTALL_DOCDIR)
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${file}
-      DESTINATION ${CMAKE_INSTALL_DOCDIR} COMPONENT doc)
-  endif()
-endforeach()
